dist/manifestor.js

```diff
@@ -2705,7 +2705,7 @@ var manifestor = function(options) {
       renderLayout(layout.intermediate(), true, endCallback);
```

Comment:
Yep I think this may be an error. I think the issue is with you including changes from master in this commit e6ebbd470f2fd18353425b13180747319063ad39. Rather than doing that, master changes should be rebased in accordingly and not included in a separate commit.